coordinate.cpp

```diff
@@ -31,6 +32,12 @@ bool coordinate::operator!=(coordinate in_coord) {
   return not((m_x == in_coord.m_x) && (m_y == in_coord.m_y));
 }
 
+std::ostream& operator << (std::ostream &out, const coordinate &cor)
+{
+    return out << cor.get_x() << " " << cor.get_y() << std::endl;
```

Comment:
I didn't know you could do this in a one-liner.

Note that the `\n` is a bad idea, as it forces the user of `operator<<` to use a newline and -sometimes- removing it. I'll create an Issue to fix that :-)